models/engine/db_storage.py

```diff
@@ -80,3 +80,24 @@ def delete(self, obj=None):
     def close(self):
         """Dispose of current session if active"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Retrieve an object"""
+        if cls is not None and type(cls) is str and id is not None and\
+           type(id) is str and cls in name2class:
+            cls = name2class[cls]
+            result = self.__session.query(cls).filter(cls.id == id).first()
+            return (result)
+        else:
+            return(None)
```

Comment:
Parentheses is literally right next to the return statement, might cause pep8 issue